.github/CODEOWNERS

```diff
@@ -8,6 +8,7 @@
 
 # DevOps for Actions and other workflow changes.
 .github/workflows @bitwarden/dept-devops
+util/MsSqlMigratorUtility/Dockerfile @bitwarden/dept-devops
```

Comment:
🎨 Check out https://github.com/bitwarden/mothership/blob/main/.github/CODEOWNERS#L13-L17 -- I think this is a stronger change.

Reply:
Thanks for the suggestion 😃